website/views/shared/tasks/edit/dailies/repeat_options.jade

```diff
@@ -1,4 +1,5 @@
-legend.option-title=env.t('repeatEvery')
+legend.option-title
+  span.hint(popover-trigger='mouseenter', popover-title=env.t('repeatEveryHelpTitle'), popover='{{env.t((task.frequency==="daily") ? "repeatDailyHelpContent" : "repeatWeeklyHelpContent")}}')=env.t('repeatEvery')
```

Comment:
I chose a ternary operator here (`(task.frequency==="daily") ? "repeatDailyHelpContent" : "repeatWeeklyHelpContent"`) for two reasons:
1. the value of `task.frequency` is lowercase, so translatable strings would have to be `repeatdailyHelpContent` or `repeatweeklyHelpContent`, thus breaking camel case convention
2. using `'repeat' + task.frequency + 'HelpContent'` makes it harder to maintain those strings in the future, in my experience